Dockerfile

```diff
@@ -1,19 +1,18 @@
-FROM node:14.20-alpine3.15
+FROM ubuntu:20.04
```

Comment:
The changes in this file are not needed anymore. Let's revert before merging the PR.